Modules/_xxsubinterpretersmodule.c

```diff
@@ -627,6 +627,36 @@ _channelends_associate(_channelends *ends, int64_t interp, int send)
     return 0;
 }
 
+static int64_t *
+_channelends_list_interpreters(_channelends *ends, int64_t *count, int send)
```

Comment:
As noted earlier, I recommend `recv` instead of `send`, though it  really doesn't matter much. :)